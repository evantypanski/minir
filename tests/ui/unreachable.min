func main() -> none {
  debug(1);
  unreachable;
}
