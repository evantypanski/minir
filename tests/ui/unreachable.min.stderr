error(interpreter): Found error while interpreting: error.Unreachable
error(interpreter): Fatal error!
